gui.py

```diff
@@ -80,17 +80,18 @@ def __init__(
             cpt["config"][-3] = cpt["weight"]["emb_g.weight"].shape[0]  # n_spk
             self.if_f0 = cpt.get("f0", 1)
             self.version = cpt.get("version", "v1")
-            if version == "v1":
-                if if_f0 == 1:
+            
+            if self.version == "v1":
+                if self.if_f0 == 1:
                     self.net_g = SynthesizerTrnMs256NSFsid(
-                        *cpt["config"], is_half=config.is_half
+                        *cpt["config"], is_half=True
```

Comment:
Why only True?
Some devices are not supported half
Please check config.py again!

Reply:
Yes. If you want to fix a wrong `False` under your GPU, just edit `config.py`.